src/bika/lims/browser/worksheet/views/printview.py

```diff
@@ -482,6 +482,7 @@ def _sample_data(self, sample):
                         sample.getDateSampled(), long_format=True),
                     'date_received': self.ulocalized_time(
                         sample.getDateReceived(), long_format=0),
+                    'client_sampleid': sample.getClientSampleID(),
```

Comment:
Note that if is an object of [`ReferenceSample`](https://github.com/senaite/senaite.core/blob/2.x/src/bika/lims/content/referencesample.py) type, this line will fail. Please add `client_sampleid` below, at line 493